docs/DeveloperGuide.md

```diff
@@ -661,16 +732,33 @@ testers are expected to do more *exploratory* testing.
 
    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
```

Comment:
just a question, does the timestamp refer to the red messages in the cmd, or the one shown in the result box?

Reply:
Honestly i wasnt too sure so i just deleted it haha

Reply:
alright I think I delete from sort as well